dask_expr/_collection.py

```diff
@@ -5099,17 +5099,18 @@ def from_dask_array(x, columns=None, index=None, meta=None):
     return from_legacy_dataframe(df, optimize=True)
 
 
+@dataframe_creation_dispatch.register_inplace("pandas")
 def read_csv(
     path,
     *args,
     header="infer",
     dtype_backend=None,
     storage_options=None,
+    _legacy_dataframe_backend="pandas",
```

Comment:
I'm open to other ideas. We can copy this entire function into dask-cudf if necessary, but it would be nice to have a simple hook like this while the `read_csv` logic is still so simple.

Reply:
I'd prefer not having a top-level argument for uses like this

Reply:
Fair enough. Removed it (Dask cuDF can just copy over the boilerplate code).